README.md

```diff
@@ -43,6 +43,64 @@ Usage: yrg [action] [args..]   load and run yara rules inside r2
 [0x100003a84]>
 ```
 
+### Yara generator usage
+
+Commands Overview
+
+    yrg - Initialize a YARA rule.
+    yrgs - Add strings as patterns.
+    yrgx - Add hex patterns.
+    yrgf - Add function byte signatures.
+    yrgz - Add all strings from the current function.
+
+To start using r2yara to create YARA rules automatically, follow these steps:
+
+##### Open a binary with radare2:
```

Comment:
dont use titles only for enumerating, 5 subtitltes is too much. better just use bold